install/share/unbound.conf.template

```diff
@@ -0,0 +1,7 @@
+server:
```

Comment:
based on the following requirement:

**DNS Root Hints**
By default, the BIND DNS server will use a set of pre-defined root server hints if it is unable to resolve a domain
name. In this situation, the server might bypass agency DNS or Protective DNS protections and contact the
DNS root servers directly. To disable this functionality, the BIND DNS server needs to be configured as a
Forwarding DNS server instead of a Recursive Name server. This can be accomplished by setting the recursion
flag to “no” in the BIND configuration.

So it would be good if we add an explicit "forward-first: no", even if this setting is the default.

Reply:
thanks, added.